flaml/automl.py

```diff
@@ -1208,9 +1226,11 @@ def _select_estimator(self, estimator_list):
                 gap = search_state.best_loss - self._state.best_loss
                 if gap > 0 and not self._ensemble:
                     delta_loss = (search_state.best_loss_old - 
-                                 search_state.best_loss)
+                                 search_state.best_loss) or \
+                                     search_state.best_loss
                     delta_time = (search_state.total_time_used - 
-                                 search_state.time_best_found_old)
+                                 search_state.time_best_found_old) or (
+                                     search_state.total_time_used + 1e-10)

```

Comment:
is this correct? in this case the 'real' delta_time, i.e., (search_state.total_time_used - search_state.time_best_found_old) is 0, do we really want to set delta_time to be search_state.total_time_used + 1e-10? shouldn't set it as 1e-10?

Reply:
Thanks. Fixed.